Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -2180,6 +2182,33 @@ int CLuaElementDefs::SetElementCollidableWith ( lua_State* luaVM )
 }
 
 
+int CLuaElementDefs::SetElementCollidableWithType ( lua_State* luaVM ) {
+    // bool SetElementCollidableWithType ( element theElement, string withType, bool enabled, bool onlyWithCreated = false )  
```

Comment:
It should begin with a lower-case **s** here and the optional argument should be surrounded with [ ] brackets:
`bool setElementCollidableWithType ( element theElement, string withType, bool enabled [, bool onlyWithCreated = false  ] )`